pxr/usd/usd/object.h

```diff
@@ -626,6 +626,30 @@ class UsdObject {
     USD_API
     bool HasAuthoredDocumentation() const;
 
+    /// Return this property's display name (metadata).  This returns the
```

Comment:
Here and below, appearances of "property" should now be "object"

Reply:
Modified comments to read `object's` instead of `property's`